akka-actor-typed-tests/src/test/scala/akka/actor/typed/LocalActorRefProviderLogMessagesSpec.scala

```diff
@@ -68,18 +68,18 @@ class LocalActorRefProviderLogMessagesSpec
       }
     }
 
-    // "logs on dedicated 'serialization' logger for empty paths" in {
-
-    //   val provider = system.asInstanceOf[ActorSystemAdapter[_]].provider
-    //   val invalidPath = provider.rootPath
-
-    //   LoggingTestKit
-    //     .debug("Resolve (deserialization) of empty path")
-    //     .withLoggerName("akka.actor.LocalActorRefProvider.Deserialization")
-    //     .expect {
-    //       provider.resolveActorRef(invalidPath.toString)
-    //     }
-    // }
+    "logs on dedicated 'serialization' logger for empty paths" in {
+
+      val provider = system.asInstanceOf[ActorSystemAdapter[_]].provider
+      val invalidPath = provider.rootPath
+
+      LoggingTestKit
+        .debug("Resolve (deserialization) of empty path")
+        .withLoggerName("akka.actor.LocalActorRefProvider.Deserialization")
+        .expect {
+          provider.resolveActorRef(invalidPath.toString)
```

Comment:
On the other hand, that sounds weird. I don't expect the root path to be deserialized.

Reply:
Actually, the only two possible situations that we can reach that code path is when the `path.root` equals the provider's `rootPath`. And the case in which we can reach the if condition is when we pass a the actor root path without any path element. 

Maybe we could even remove that condition.